### DISCUSSION TITLE
Purpose and definition of OMR default LAN "Override IPv4 routing table" "lan (50)"?

### DISCUSSION THREAD

**Discussion by luckygreen** [Ysurac/openmptcprouter#2698] [Q&A]
Out-ouf-the-box, OMR's default LAN is configured with an "Override IPv4 routing table" of "lan (50)".

1)  Why is this "Override IPv4 routing table" required? I.e., what is its purpose?
2)  Where in the GUI can I view which specific routing table overrides "lan (50)" performs?
I.e., where are the contents of "lan (50)" defined?

Thanks!

**Comment by Ysurac** [comment]
1. ```ip r show table 50``` will give you default table with 192.168.100.0/24 dev eth0...
2. As it's interface routing table it override nothing and will always be interface subnet.

**Reply by luckygreen** [reply]
In which config files or scripts are these tables defined? I looked through the config files, but can't seem to be able to locate the source of the various table entries?

I am still struggling to understand why those tables are necessary  and my hope is that when I see their source in context their purpose will become clearer to me. Thanks!
